sacred/utils.py

```diff
@@ -677,17 +676,17 @@ def module_is_imported(modname, scope=None):
     return False
 
 
-def get_package_version(dist_name):
-    """Returns a parsed version string of a package."""
-    version_string = pkg_resources.get_distribution(dist_name).version
-    return version.parse(version_string)
-
-
 def parse_version(version_string):
     """Returns a parsed version string."""
     return version.parse(version_string)
 
 
+def get_package_version(name):
+    """Returns a parsed version string of a package."""
+    version_string = __import__(name).__version__
```

Comment:
Python docs recommend to use `importlib.import_module` over `__import__`, see https://docs.python.org/3/library/functions.html#__import__ Is there anything that speaks against using `import_module` here?

Reply:
I couldn't find any reason why not to use the `importlib`. I tested it successfully. Thank you for the tip. I will push the changes shortly.